portafly/src/i18n/locales/en/accounts.yml

```diff
@@ -1,20 +1,95 @@
-  col_admin: Admin
-  col_admin_description: Title of the Admin column
-  col_signup: Signup date
-  col_signup_description: Title of the Signup date column
-  col_apps: Apps
-  col_apps_description: Title of the Apps column
-  col_state: State
-  col_state_description: Title of the State column
+# Account settings > Users
+
+title_page: Accounts | Team members
+title_page_description: Title for the page to manage team members
+
+# Top area
+
+body_title: Team members
+body_title_description: Title for the main area to manage team members
+invite_new_member_button: Invite a new team member
+invite_new_member_button_description: Button link to send invitations to new team members
+
+# Main area
+
+## Tabs
+listing_tab: Listing
+listing_tab_description: Label for the tab that contains the list of all team members, selection criteria and actions that can be applied on them.
+invitations_tab: Invitations
+invitations_tab_description: Label for the tab that contains the list of all invitations, selection criteria and actions that can be applied on them.
+
+## Tab: Listing
+## Toolbar
+bulk_selector: {{number}} selected
+bulk_selector_description: Label for the bulk selector to indicate the number of chosen team members. This label is visible only when team members were selected.
+# Delete button: available in shared.yml
+team_members_filter: [Name]
+team_members_filter_description: List containing the criteria to filter team members
+[PROPOSED] team_members_typeahead: Filter by name
+team_members_typeahead_description: Filter by typing ahead the text
+
+team_members_table:
+  aria_label: Team members table
+  aria_label_description: Aria label of the table of team members
+  [PROPOSED] empty_state: Currently, there are no team members. Add team members.
+  empty_state_description: Message to show when there are no team members to show, and prompt the user to add members
+  name_header: Name
+  name_header_description: Column listing names of team members
+  email_header: Email
+  email_header_description: Column listing the email addresses of team members
+  sso_authentication_heaer: SSO authentication
+  sso_authentication_header_description: Column listing if SSO authentication has been configured for team members
+  [PROPOSED] sso_authentication_value: [Yes, Not yet]
+  role_header: Role
+  role_header_description: Column listing the roles associated to each team member
+  [PROPOSED] role_header_value: [Admin (you), Admin, Member]
+  permission_groups_header: Permission groups
+  permission_groups_header_description: Column listing the permisions mapped to each team member
+
+## Tab: Invitations
+## Toolbar
+bulk_selector: {{number}} selected
+bulk_selector_description: Label for the bulk selector to indicate the number of chosen invitations. This label is visible only when invitations were selected.
+#-------# Delete button: available in shared.yml
+[PROPOSED] invitations_filter: [Recipient, Acceptance]
```

Comment:
In the cases where we have several options like [a, b, c] it's better to have a single entry for every option, in this way we don't need to implement logic to extract the value.
Only when the literal value to show is `[a, b, c]` it's OK to have a single entry, but when those are possible values (or a, or b, or c) is better having an entry for each option

Reply:
Pending answer from #1861 (comment)